examples/autobahn-client.rs

```diff
@@ -31,7 +33,7 @@ fn run_test(case: u32) -> Result<()> {
     let case_url = Url::parse(
         &format!("ws://localhost:9001/runCase?case={}&agent={}", case, AGENT)
     ).unwrap();
-    let (mut socket, _) = connect(case_url)?;
+    let (mut socket, _) = connect(case_url, None)?;
```

Comment:
Yes, we could make 2 set of functions, the first set of functions does not take any parameters and works with the default values, while the second set of functions takes a config.

NB: In this case, we still have to raise the version as the changes are anyway not compatible (the result of the `write_message()` function has changed its signature as now it may return the message back when the queue is happened to be full).

Reply:
Do I have to make this change? (we will anyway have to release a new version and it will not be fully backward-compatible as mentioned in my message, but if needed I can create a helper `connect()` and other functions which use the default config)